website/blog/2019-06-04-hello-reason-react-native.md

```diff
@@ -0,0 +1,129 @@
+---
+id: 2019-06-04-hello-reason-react-native
+title: Hello Reason React Native
+---
+
+Today is a big day for the ReasonML community as we are releasing the fruits of
+a long effort started 3 months ago. As more and more people were starting to use
```

Comment:
`Like a growing number of developers, we started using ReasonML on production apps using React Native and we wanted…`